src/Evolution/Systems/Cce/ScriPlusValues.cpp

```diff
@@ -254,10 +254,11 @@ void CalculateScriPlusValue<Tags::ScriPlus<Tags::Psi2>>::apply(
       -0.5 * get(boundary_r) *
       (-exp_2_beta_at_scri * (conj(ethbar_dy_q_at_scri) +
                               eth_r_divided_by_r_view * conj(dy_q_at_scri)) +
-       get(boundary_r) * (ethbar_dy_dy_u_at_scri +
-                          conj(eth_r_divided_by_r_view) * dy_dy_u_at_scri +
-                          conj(ethbar_dy_dy_u_at_scri) +
-                          eth_r_divided_by_r_view * conj(dy_dy_u_at_scri)) +
+       get(boundary_r) *
+           (ethbar_dy_dy_u_at_scri +
+            2.0 * conj(eth_r_divided_by_r_view) * dy_dy_u_at_scri +
```

Comment:
is the documentation correct ?

Reply:
Yeah, the documentation leaves the Jacobian factors implicit, so doesn't need to change based on this realization.